coral-component-list/src/scripts/SelectList.js

```diff
@@ -526,7 +526,7 @@ const SelectList = Decorator(class extends BaseComponent(HTMLElement) {
 
     // adds the role to support accessibility
     if (!this.hasAttribute('role')) {
-      this.setAttribute('role', 'listbox');
+      this.setAttribute('role', 'list');
```

Comment:
This change should not be made to the SelectList control. The WAI-ARIA `listbox` role is a widget role, that will force a screen reader into forms mode, allowing the user to navigate using the arrow keys. A `list` is a static role defining content structure. Think about this as the difference between a `<select>` element and a `ul` or `ol` in HTML.